ros2action/package.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>ros2action</name>
+  <version>0.6.3</version>
```

Comment:
should the version number be reset here?

Reply:
I'm not sure, but I think for the purpose of tagging releases for all packages in the repo at once it is more convenient to keep the same version across packages. Otherwise, there's the issue of associating Git release tags and the different versions across packages.